frontend/src/styles.scss

```diff
@@ -41,3 +41,28 @@ ngb-datepicker-navigation button {
   border: none;
 }
 
+.footer-chuck {
+  height: 14px;
+  vertical-align: text-bottom;
+}
+
+/* Sticky footer styles
```

Comment:
See comment in app.component.html: all these styles are not needed.

Reply:
OK I delete them